tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -101,6 +101,39 @@ task ExtractUMIs {
   }
 }
 
+# Adapter clipping
+task Fastp {
+  input {
+    File fastq1
+    File fastq2
+    String output_prefix
+    File adapter_fasta = "gs://broad-dsde-methods-takuto/RNA/resources/Illumina_adapters.fasta" # sato: move to a public place
+  }
+
+  Int disk_size = 5*ceil(size(fastq1, "GiB")) + 128
+
+  command {
+    fastp --in1 ~{fastq1} --in2 ~{fastq2} --out1 ~{output_prefix}_read1.fastq.gz --out2 ~{output_prefix}_read2.fastq.gz \
+    --disable_quality_filtering --adapter_fasta ~{adapter_fasta} 
+  }
+  
+
+  runtime {
+    docker: "biocontainers/fastp:v0.20.1_cv1"
+    memory: "8 GiB"
+    disks: "local-disk " + disk_size + " HDD"
+    preemptible: 0
+  }
+
+  output {
+    File fastq1_clipped = output_prefix + "_read1.fastq.gz"
+    File fastq2_clipped = output_prefix + "_read2.fastq.gz"
+    File html = "fastp.html" # sato: these are optional
```

Comment:
Not sure what is meant by these comments.

Reply:
removed